docs/source/admin/quick_howto/ciab.rst

```diff
@@ -376,7 +376,7 @@ On Linux, we suggest ``openvpn``. On most Linux distributions, this will also be
 .. code-block:: shell
 	:caption: Install openvpn on ubuntu/debian
 
-	apt-get update && apt-get install -y openvpn
+	apt-get update && apt-get --no-install-recommends install -y openvpn
```

Comment:
This is just for people to install a vpn client on the host, so I don't think there's any need for this.

Reply:
I general updated "apt-get install" with "apt-get --no-install-recommends install" as a best practice for apt-get wherever applicable

Reply:
Right, but in this case the user is installing something on their own computer. `--no-install-recommends` is a "best practice" for the context of docker, but ostensibly the user has the default apt configuration they want on their own host computer. We don't need to worry - and shouldn't worry - about policing that.